src/rt/dwarfeh.d

```diff
@@ -73,6 +73,11 @@ else version (MIPS32)
     enum eh_exception_regno = 4;
     enum eh_selector_regno = 5;
 }
+else version (RISCV64)
```

Comment:
Perhaps, shouldn't be available a `regno` for `RISCV32` version?

Reply:
It seems to be the same, according to this LLVM diff https://reviews.llvm.org/D63411

Reply:
Thanks for your review. I'm currently working on riscv64gc native environment, and I don't have a native riscv32 gcc available now to execute the test code in the comment above. Maybe someone more experienced could help?

Reply:
It seems to be correct, according to both LLVM and RISC-V ABI docs. See https://github.com/riscv-non-isa/riscv-elf-psabi-doc/blob/master/riscv-dwarf.adoc#dwarf-register-numbers . `X10` and `X11` maps directly to DWARF registers, so it should be good for both architectures according to the LLVM patch. Also, according to the comments on that diff, they based those values on GCC, so it should match, but the RISC-V ABI documentation should mention it explicitly. I can file an issue on it to clarify those values.

Reply:
Cross referencing: https://github.com/riscv-non-isa/riscv-elf-psabi-doc/issues/293